lib/core.sh

```diff
@@ -170,7 +170,7 @@ function _get_remote_command(){
 [[ -w "$KYRAT_TMPDIR" ]] || { echo >&2 "Could not write into temp directory $KYRAT_TMPDIR on the remote host. Set KYRAT_TMPDIR env variable on a writable remote directory. Aborting."; exit $NO_WRITABLE_DIRECTORY; };
 command -v $BASE64 >/dev/null 2>&1 || { echo >&2 "kyrat requires $BASE64 command on the remote host. Aborting."; exit $NOT_EXISTING_COMMAND; };
 command -v $GUNZIP >/dev/null 2>&1 || { echo >&2 "kyrat requires $GUNZIP command on the remote host. Aborting."; exit $NOT_EXISTING_COMMAND; };
-kyrat_home="\$(mktemp -d kyrat-XXXXX -p "$KYRAT_TMPDIR")";
+kyrat_home="\$(mktemp -d "/$KYRAT_TMPDIR/kyrat-XXXXX")";
```

Comment:
I think you can remove the leading slash. `KYRAT_TMPDIR` should always be an absolute path:

```suggestion
kyrat_home="\$(mktemp -d "$KYRAT_TMPDIR/kyrat-XXXXX")";
```